src/main/webapp/app/shared/components/documentation-button/documentation-button.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input } from '@angular/core';
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons';
+
+// The routes here are used to build the link to the documentation.
+// Therefore, it's important that they exactly match the url to the subpage of the documentation.
+// Additionally, the case names must match the keys in documentationLinks.json for the tooltip.
+export enum DocumentationType {
+    Course = <any>'courses/customizable/',
+    Lecture = <any>'lectures/',
+    Exercise = <any>'exercises/',
+    Quiz = <any>'exercises/quiz/',
+    Model = <any>'exercises/modeling/',
+    Programming = <any>'exercises/programming/',
+    Text = <any>'exercises/textual/',
+    FileUpload = <any>'exercises/file-upload/',
+    Notifications = <any>'notifications/',
+    LearningGoals = <any>'lectures/#learning-goals',
+    Communications = <any>'communication/',
+    Exams = <any>'exam_mode/',
+    PlagiarismChecks = <any>'plagiarism-check/',
+    Grading = <any>'grading/',
+    Units = <any>'lectures/#lecture-units',
+    Assessment = <any>'exercises/assessment/',
+    Statistics = <any>'learning-analytics/',
+}
+
+@Component({
+    selector: 'jhi-documentation-button',
+    styleUrls: ['./documentation-button.component.scss'],
+    template: `
+        <button type="button" class="text-primary documentation-button" (click)="openDocumentation()">
+            <fa-icon [icon]="faCircleInfo" ngbTooltip="{{ getTooltipForType() | artemisTranslate }}"></fa-icon>
```

Comment:
Can you use our `jhi-help-icon.component` here?

Reply:
It's not the same design so I would not use it. Also in both cases it's 1 line so I think it's fine with using my own icon